src/components/CommentsList.jsx

```diff
@@ -35,15 +52,46 @@ export default function CommentsList({ articleId }) {
             .slice(0, count)}
         </div>
       )}
-      {count < comments.length ? (
-        <button
-          onClick={() => {
-            handleClick();
-          }}
-        >
-          See more
-        </button>
-      ) : null}
+
+      <>
+        {userAuthorized ? (
+          <form
+            onSubmit={(e) => {
+              addComment(e);
+            }}
+          >
+            <label htmlFor="comment">Write comment:</label>
+            <input
+              type="text"
+              name="comment"
+              id="comment"
+              placeholder="Add a comment"
+              onChange={(e) => {
+                setComment(e.target.value);
+              }}
+              value={comment}
+            />
+            <div className="buttons">
+              <button type="reset" onClick={() => setComment("")}>
+                Cancel
+              </button>
+              <button type="submit" disabled={comment === "" ? true : false}>
```

Comment:
You could also make the comment input required to achieve this effect without using `disabled`. Also, a comment may be quite lengthy so a `textarea` is often a nicer tag to use